test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1683,6 +1683,8 @@ var Annotations = map[string]string{
 
 	"[sig-olmv1] OLMv1 Catalogs should be installed": " [Suite:openshift/conformance/parallel]",
 
+	"[sig-olmv1] OLMv1 operator installation should block cluster upgrades if an incompatible operator is installed": " [Suite:openshift/conformance/parallel]",
```

Comment:
This is a bit concerning to me... as it's implying that these could be running in parallel... so you might be trying to run your two tests _at the same time_, which would give unpredictable results. You may need to combine the tests into a single `g.It()` clause.

Reply:
If that avoids the parallelism, then that's ok.

Reply:
I see it in other sequential jobs.

From https://onsi.github.io/ginkgo/#ordered-containers:

> Specs in an Ordered container may run in parallel with respect to other specs, but they will always run sequentially on the same parallel process.


Should be good enough for this test I think.